modules/fundamental/src/purl/model/details/purl.rs

```diff
@@ -36,7 +40,15 @@ pub struct PurlDetails {
     pub version: VersionedPurlHead,
     pub base: BasePurlHead,
     pub advisories: Vec<PurlAdvisory>,
-    pub licenses: Vec<PurlLicenseSummary>,
+    pub licenses: Vec<LicenseInfo>,
+    pub license_ref_mapping: Vec<LicenseRefMapping>,
```

Comment:
Please rename this to `licenses_ref_mapping` to be exactly the same as the one current returned from the SBOM packages endpoint, ref. https://github.com/guacsec/trustify/blob/3818788ca7f893d3d9bfa03bd71b03216017a857/modules/fundamental/src/sbom/model/mod.rs#L135

Reply:
You did partially because, as I wrote in the review above, also the `test_purl_license_details` test must be changed accordingly and, in fact, that test fails with the last commit.
Please always remember to run affected tests locally before pushing the code.

Reply:
ok